pom.xml

```diff
@@ -98,6 +98,13 @@
         <version>${version.org.jgrapht}</version>
       </dependency>
 
+      <!-- servlet dependencies -->
+      <dependency>
+        <groupId>org.jboss.spec.javax.servlet</groupId>
+        <artifactId>jboss-servlet-api_3.0_spec</artifactId>
```

Comment:
Just curious why `jboss-servlet-api_3.0_spec` is used and not `jboss-servlet-api_3.1_spec` ?
Nevertheless this is probably of no importance in this case.